R/table-subscript.R

```diff
@@ -641,6 +649,20 @@ addArrayIndicesIfMissing <- function(q.test.info, y, dim.names, qtypes, sep = "_
     return(cbind(arr.idx, q.test.info))
 }
 
+checkArrayIndices <- function(dim.names, q.test.indices)
```

Comment:
Could perhaps improve the function name here? Maybe `qTableDimnamesMatchQStatInfo`?

Reply:
Ideally have just the `qtable` as the input to the function. It'll interrogate the `dimnames` and q stat testing info and return `TRUE` or `FALSE`.

Reply:
I've renamed it to `qTableDimnamesMatchQStatInfo`, but it can't take `qtable` as input because its called inside `addArrayIndicesIfMissing` so the table is not available to it